packages/account-management/src/frontend/Shared/ErrorPage.tsx

```diff
@@ -3,49 +3,66 @@ import SpacingComponent from '@weco/common/views/components/SpacingComponent/Spa
 // @ts-ignore
 import SectionHeader from '@weco/common/views/components/SectionHeader/SectionHeader';
 import { OutlinedButton } from '@weco/common/views/components/ButtonOutlined/ButtonOutlined';
+import { useLocationQuery } from '../../hooks/use-location-query';
+
 import styled from 'styled-components';
-import { useHistory } from 'react-router-dom';
 
 // TODO: Update this to prod.
 const logo = 'https://identity-public-assets-stage.s3.eu-west-1.amazonaws.com/images/wellcomecollections-150x50.png';
 
 const LogoContainer = styled.div`
-   {
-    margin: auto;
-    padding: 42px;
-    width: 200px;
-  }
+  margin: auto;
+  padding:  0 0 42px 0;
+  width: 200px;
+  border-bottom: 0.2px solid grey;
 `;
 
-const PageContainer = styled.div`
-   {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-  }
+const Auth0StyleErrorBox = styled.div`
+  background-color: white;
+  padding: 42px;
+  margin: auto;
+  border-radius: 5px;
+  height: 400px;
+  width: 300px;
+  justify-content: space-between;
+  display: flex;
+  flex-direction: column;
 `;
 
-export const ErrorPage = () => {
-  const history = useHistory();
+const PageContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #f0ede3;
+  padding: auto;
+  height: 100%;
+`;
 
-  const redirectToWellcomeHelpPage = () => {
-    // Is it possible to configure the support url through auth0.
-    history.push('/support');
-  };
+const ErrorMessage = styled.p`
+  text-align: center;
+`;
 
+export const ErrorPage = () => {
+  const { error_description } = useLocationQuery();
```

Comment:
`useLocationQuery` can take a generic type parameter. Should we provide one here?

```ts
type ErrorQueryParams = {
  error_description: string | undefined;
}

// ...

const { error_description } = useLocationQuery<ErrorQueryParams>();
```

I haven't tested this, and there's probably a better name than `ErrorQueryParams`